darts/models/forecasting/torch_forecasting_model.py

```diff
@@ -1244,6 +1244,138 @@ def lr_find(
             update_attr=False,
         )
 
+    @random_method
+    def scale_batch_size(
+        self,
+        series: Union[TimeSeries, Sequence[TimeSeries]],
+        val_series: Union[TimeSeries, Sequence[TimeSeries]],
```

Comment:
I think we can remove all `val_*` arguments, since we use the same batch size for training and evaluation.

For the `val_dataloader` in the `DataModule`, we can just use `series`, `past_covariates`, `future_covariates` for the input dataset